cli/names.go

```diff
@@ -0,0 +1,19 @@
+package cli
+
+import (
+	"strings"
+
+	"github.com/docker/cli/cli/names"
+)
+
+var (
+	validContainerNamePattern = names.RestrictedNamePattern
+)
+
+// CheckContainerName check container's name is valid or not
+func CheckContainerName(name string) bool {
+	if len(name) == 0 {
```

Comment:
Why checking the length? I think the regular expression won't match an empty string.

Reply:
Yes, thanks.